### PR TITLE
UC Browser Cheat Sheet

### PR DESCRIPTION
**Pull request by robinjoseph070**
DDG Community Kochi
#2127 
https://duck.co/ia/view/uc_browser_cheat_sheet

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
cc @moollaza 

He didn't received comment from dax.
I will mail his details.

**Comment by moollaza** [comment]
@robinjoseph070 @gautamkrishnar the id in the file is incorrect -- that may be part of the problem

File: share/goodie/cheat_sheets/jsonucbrowser.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+  "id": "adobe_brackets_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/jsonucbrowser.json]: @robinjoseph070 I see you copied an existing cheat sheet? Don’t forget to change all the data accordingly!

**Comment by moollaza** [comment]
@robinjoseph070 Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by gautamkrishnar** [comment]
Finishing it myself

**Comment by edgesince84** [comment]
@gautamkrishnar,  thanks again for creating  the UC Mobile Cheat Sheet! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by mintsoft** [comment]
Already implemented in: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2704

**Pull request closed without merging**